bootstrap/web/iframe-adapter.js

```diff
@@ -296,6 +299,10 @@ var ZoweZLUX = {
     }
 }
 
+ZoweZLUX.logger = new exports.Logger();
+ZoweZLUX.logger.addDestination(ZoweZLUX.logger.makeDefaultDestination(true, true, true))
```

Comment:
You need some more true here. `logger.addDestination(logger.makeDefaultDestination(true,true,true,true,true));`